ee/ucp/admin/configure/deploy-route-reflectors.md

```diff
@@ -1,137 +1,133 @@
 ---
-title: Improve network performance with Route Reflectors
-description: Learn how to deploy Calico Route Reflectors to improve performance
-  of Kubernetes networking
+title: Improve network performance with route reflectors
+description: Learn how to deploy Calico route reflectors to improve the performance of Kubernetes networking.
 keywords: cluster, node, label, certificate, SAN
 ---
 
 >{% include enterprise_label_shortform.md %}
 
-UCP uses Calico as the default Kubernetes networking solution. Calico is
-configured to create a BGP mesh between all nodes in the cluster.
+Docker Universal Control Plane (UCP) uses [Calico](https://docs.projectcalico.org/v3.11/introduction/) as the default Kubernetes networking solution. Calico is configured to create a Border Gateway Protocol (BGP) mesh between all nodes in the cluster.
 
 As you add more nodes to the cluster, networking performance starts decreasing.
 If your cluster has more than 100 nodes, you should reconfigure Calico to use
-Route Reflectors instead of a node-to-node mesh.
+route reflectors instead of a node-to-node mesh.
 
-This article guides you in deploying Calico Route Reflectors in a UCP cluster.
+Route reflectors are useful in large scale deployments to reduce the number of BGP connections needed for correct and complete route propagation.
+
+This topic guides you in deploying Calico route reflectors in a UCP cluster.
```

Comment:
In my opinion, this sentence is not necessary as the purpose of the topic is covered in the title (or should be...perhaps a better title would be **Deploying Calico route reflectors to improve network performance**).

If you do opt to drop the sentence and/or change the title, the Azure point should be offered as a Note:

*Note:*

>UCP instances running on Microsoft Azure use Azure SDN for multi-host networking, and not Calico. Refer to [Install UCP on Azure](https://docs.docker.com/ee/ucp/admin/install/cloudproviders/install-on-azure/) for more information).